### PR TITLE
Feature/website

### PR DESCRIPTION
**Pull request by ptim**
@brenton @darren @fabio

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) HTML5 Boilerplate
```
Comment (MarkusH) [inline LICENSE.txt]: I highly doubt they have the © for this website. Maybe for the boilerplate, but not more.

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (MarkusH) [inline Readme.md]: Chop empty lines at beginning of files.

**Comment by MarkusH** [comment]
There are a few .map files committed. I don't think we need them in the repo, do we?

File: LICENSE.txt (hunk 1 above)
Comment (ptim) [inline LICENSE.txt]: heh, this is a todo for @sesh :)

File: assets/commoncode-logo.svg [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (ptim) [inline assets/commoncode-logo.svg:1]: should optimise these svgs

**Pull request merged**